app/src/main/java/np/com/naxa/factsnepal/feed/list/FeedListActivity.java

```diff
@@ -104,11 +105,11 @@ protected void onCreate(Bundle savedInstanceState) {
 //        setupChips(null);
         initChips();
         setupNavigationDrawer();
-        
-               cardSurvey.setOnClickListener(new View.OnClickListener() {
+
+        cardSurvey.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                ActivityUtil.openActivity(PublicPollActivity.class,FeedListActivity.this,null,false);
+                ActivityUtil.openActivity(PublicPollActivity.class, FeedListActivity.this, null, false);
```

Comment:
Add the new function in `ActivityUtil` `openActivity(PublicPollActivity.class, FeedListActivity.this){}`